bskyweb/cmd/bskyweb/server.go

```diff
@@ -223,9 +240,9 @@ func serve(cctx *cli.Context) error {
 	e.GET("/profile/:handleOrDID/post/:rkey/liked-by", server.WebGeneric)
 	e.GET("/profile/:handleOrDID/post/:rkey/reposted-by", server.WebGeneric)
 
-    // starter packs
```

Comment:
mfw i have to remember to use a tab 😅 thanks

Reply:
When in doubt you can always use `go fmt`!